workflow/tests/test_workflow_models.py

```diff
@@ -4,12 +4,92 @@
 from django.test import TestCase
 from workflow.models import (
     Organization, Program, Country, Province, ProjectAgreement, Sector,
-    ProjectComplete, ProjectType, SiteProfile, Office, Benchmarks, Budget
+    ProjectComplete, ProjectType, SiteProfile, Office, Benchmarks, Budget,
+    Documentation, Checklist, ProjectStatus
 )
 
+class ProjectStatusTestCase(TestCase):
+
+
+    def setUp(self):
+        new_project_status = ProjectStatus.objects.create(name="project status")
+        new_project_status.save()
+
+    def test_project_status_creation(self):
+        """Check for ProjectStatus Object creation"""
+        get_project_status = ProjectStatus.objects.get(name="project status")
+        self.assertTrue(isinstance(get_project_status, ProjectStatus))
+        self.assertTrue(get_project_status.__str__(), get_project_status.name)
```

Comment:
You probably meant to use `assertEqual` for comparing both strings

Reply:
This is actually the same, though let me change it for readability purpose